actions/types.ts

```diff
@@ -265,6 +271,7 @@ export interface Campaign {
   latestDeckIds?: number[]; // deprecated
   investigatorData: InvestigatorData;
   chaosBag: ChaosBag;
+  chaosBagResults: ChaosBagResults;
```

Comment:
So because this goes into redux (and is persisted on devices), we'll need to make it optional (`?: ChaosBagResults`) and handle giving it a default case in our code.